### PR TITLE
Add X25519 and fix CI timeouts

### PR DESCRIPTION
**Pull request by jargh**
*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: codebuild/cloudformation.yml [hunk 1]
```diff
@@ -81,10 +81,10 @@ Resources:
             - Type: ACTOR_ACCOUNT_ID
               Pattern: 829871 # SalusaSecondus / rubin@amazon.com
               ExcludeMatchedPattern: false
```
Comment (SalusaSecondus) [inline codebuild/cloudformation.yml:83]: Remove my permissions.
```suggestion
```

File: codebuild/cloudformation.yml [hunk 2]
```diff
@@ -144,10 +144,10 @@ Resources:
...
```
Comment (SalusaSecondus) [inline codebuild/cloudformation.yml:146]: Remove my permissions.
```suggestion
```

File: codebuild/cloudformation.yml [hunk 3]
```diff
@@ -213,10 +213,10 @@ Resources:
...
```
Comment (SalusaSecondus) [inline codebuild/cloudformation.yml:215]: Remove my permissions.
```suggestion
```

File: codebuild/cloudformation.yml [hunk 4]
```diff
@@ -282,10 +282,10 @@ Resources:
...
```
Comment (SalusaSecondus) [inline codebuild/cloudformation.yml:284]: Remove my permissions.
```suggestion
```

**Review by SalusaSecondus — COMMENTED** [review]
Please remove all of my permissions.

**Review by torben-hansen — APPROVED** [review]

**Pull request merged**